assignments/function-conversion.js

```diff
@@ -1,24 +1,18 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
+let myFunction = () => { console.log("Function was invoked!"); };
+myFunction();
 
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
+let anotherFunction = param => { param };
+anotherFunction("Example");
 
-// let add = function (param1, param2) {
-//   return param1 + param2;
-// };
-// add(1,2);
+let add = (param1, param2) => { return param1 + param2; };
+add(1, 2);
 
-// let subtract = function (param1, param2) {
-//   return param1 - param2;
-// };
-// subtract(1,2);
+let subtract = (param1, param2) => {
+    return param1 - param2;
+};
+subtract(1, 2);
```

Comment:
Don't forget to console.log so your data shows in the console and you can keep track out it. console.log(subtract(1,2))